### DISCUSSION TITLE
Push local files to the repository through the github desktop, but the deployment cannot be performed.

### DISCUSSION THREAD

**Discussion by jlduu** [alshedivat/al-folio#1548] [Q&A]
Could I edit my information directly on the GitHub website instead of editing it locally and pushing it to the repository? In fact, when I use github desktop to edit locally and push to the repository, the deployment always goes wrong.

**Comment by george-gca** [comment]
Have you set up properly the remote repository? Is it correctly displaying the site from the beginning?

**Reply by jlduu** [reply]
Thanks for your reply, it worked and deployed correctly at first. Now, I forked the theme from a third party and it works fine.

**Reply by george-gca** [reply]
Out of curiosity, which third party?

**Comment by jlduu** [comment]
> Out of curiosity, which third party?

https://github.com/zuobinxiong/zuobinxiong.github.io
